tool/enc-unicode.rb

```diff
@@ -161,14 +161,24 @@ def parse_scripts(data, categories)
   names = {}
   files.each do |file|
     data_foreach(file[:fn]) do |line|
+      # Parse Unicode data files and store code points and properties.
       if /^# Total (?:code points|elements): / =~ line
         data[current] = cps
         categories[current] = file[:title]
         (names[file[:title]] ||= []) << current
         cps = []
-      elsif /^(\h+)(?:\.\.(\h+))?\s*;\s*(\w+)/ =~ line
-        current = $3
+      elsif /^(\h+)(?:\.\.(\h+))?\s*;\s*(\w(?>[\w\s;]*\w)?)/ =~ line
```

Comment:
Is there a reason to use atomic group `(?>   )` here? I think `(?:   )` is enough.

Reply:
I've fixed it.